spec/system/home_page_spec.rb

```diff
@@ -1,9 +1,7 @@
 require "rails_helper"
 
 RSpec.feature "Home Page" do
-  after do
-    Capybara.app_host = nil
-  end
+  after { Capybara.app_host = nil }
```

Comment:
Did the linter do this? If so, fine (odd it wasn't picked up before). 

If it didn't, I don't think we should introduce changes to unrelated files in a PR. Both syntaxes are valid and the change doesn't have anything to do with the task, so there is no need to change it.